cmd/net.go

```diff
@@ -66,17 +67,42 @@ func mustGetLocalIP4() (ipList set.StringSet) {
 // getHostIP4 returns IPv4 address of given host.
 func getHostIP4(host string) (ipList set.StringSet, err error) {
 	ipList = set.NewStringSet()
-	ips, err := net.LookupIP(host)
-	if err != nil {
-		return ipList, err
+	//variable to list resolved IP addresses
+	var ips []net.IP
+	// wait for hosts to resolve in case of container environment.
+	// This is needed for orchestration platforms like kubernetes,
+	// as the remote hosts are sometime not available immediately.
+	if IsDocker() {
+		// channel to indicate completion of host resolution
+		doneCh := make(chan struct{})
+		// Indicate to our retry routine to exit cleanly, upon this function return.
+		defer close(doneCh)
+		// Mark the starting time
+		startTime := time.Now().Round(time.Second)
+		// Wait on the jitter retry loop
+		for _ = range newRetryTimerSimple(doneCh) {
+			// try to resolve remote host
+			ips, err = net.LookupIP(host)
+			// log error and continue. We dont fail on host resolution, just log
+			// the message to console about the host not being resolveable.
+			errorIf(err, "Unable to resolve host %s (elapsed %s)", host, time.Now().Round(time.Second).Sub(startTime).String())
```

Comment:
Please break long lines (>80 chars) wherever possible and convenient - it is easier to read. Here, the line can be broken after the host argument.